src/Templates/src/templates/maui-blazor/Properties/launchSettings.json

```diff
@@ -1,7 +1,7 @@
 {
   "profiles": {
     "Windows Machine": {
-      "commandName": "MsixPackage",
+      "commandName": "Project",
```

Comment:
Comments aren't supported in json by defauly, and the launchsettings schema doesn't add this support

Reply:
System.Text.Json has `JsonCommentHandling.Allow`, but I don't know what tooling parses this file.

Reply:
Should we delete this file since it is no longer needed? The default command for a dotnet app is "Project" and this file was just used to override that for MSIX.